niworkflows/interfaces/surf.py

```diff
@@ -83,6 +86,80 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class _Path2BIDSInputSpec(BaseInterfaceInputSpec):
+    in_file = File(mandatory=True, desc='input GIFTI file')
+
+
+class _Path2BIDSOutputSpec(DynamicTraitedSpec):
+    extension = traits.Str()
+
+
+class Path2BIDS(SimpleInterface):
```

Comment:
Is this being used?

Reply:
yes, in [fmriprep's](https://github.com/poldracklab/fmriprep/pull/2114/files#diff-928c5651a2bdc224e2a84acff159a0bcR271) and [smriprep's](https://github.com/poldracklab/smriprep/pull/183/files#diff-42a7589488f1889936965d52cf5c02b2R238) PRs